install.sh

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2020, The Pallene Developers
+# Pallene is licensed under the MIT license.
+# Please refer to the LICENSE and AUTHORS files for details
+# SPDX-License-Identifier: MIT
+
+####
+# One script to rule them all, one script to find them,
+# one script to bring them all and in the ease-of-use bind them.
+####
+
+# Versions
+LUA_VERSION=5.4.7
+LUAROCKS_VERSION=3.9.1
+PT_VERSION=0.5.0a
+
+# Where we will clone
+CLONE_DIR="/tmp/pallene"
+mkdir -p $CLONE_DIR
+
+# Current working directory
+CURR_DIR=$(pwd)
+
+# Where to install/locate Lua
+LUA_PREFIX=${LUA_PREFIX:-/usr/local}
```

Comment:
Please initialize everything that has a default value here on top. That way it's easier to see what all the configuration options are. (For example, PALLENE_LOCAL, etc)